lms/urls.py

```diff
@@ -586,6 +587,11 @@
     urlpatterns += (
         url(r'', include('third_party_auth.urls')),
         url(r'^login_oauth_token/(?P<backend>[^/]+)/$', 'student.views.login_oauth_token'),
+        url(
+            r'^exchange_oauth_token/$',
+            oauth_exchange.views.AccessTokenExchangeView.as_view(),
+            name="exchange_oauth_token"
+        ),
```

Comment:
Despite being a subdirectory of common/djangoapps, third_party_auth is in fact not a django app. It merely defines a bunch of mechanisms that are ultimately used by python-social-auth for user authentication. There is no existing endpoint namespace that I am aware of that would be appropriate for this endpoint.